kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-testing/pom.xml

```diff
@@ -522,6 +528,27 @@
             </executions>
           </plugin>
 
+
+          <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>build-helper-maven-plugin</artifactId>
+            <version>3.0.0</version>
+            <executions>
+              <execution>
+                <id>add-source</id>
+                <phase>generate-resources</phase>
+                <goals>
+                  <goal>add-source</goal>
+                </goals>
+                <configuration>
+                  <sources>
+                    <source>/Volumes/Navi/Projects/forks/kogito-editors-java/kogito-editors-js</source>
```

Comment:
Is this correct?

Reply:
Thanks, @manstis! I missed this directory here. I'm fixing in a separate commit. Thanks for checking it! :-)

Reply:
Actually, I've applied the fix in a squashed commit ;-) Thanks again for checking, Mike!